systemtest/src/test/java/io/strimzi/systemtest/timemeasuring/Operation.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+// Original author: David Kornel, EnMasse
+
+package io.strimzi.systemtest.timemeasuring;
+
+public enum Operation {
```

Comment:
Do we really need to have this enum? because if I'm not wrong we use only TEST_EXECUTION in ST. Other enums are not used.